readthedocsext/theme/templates/organizations/partials/organization_list.html

```diff
@@ -66,7 +65,11 @@
 {% endblock list_item_right_buttons %}
 
 {% block list_item_image %}
-  <img class="ui image" src="{% gravatar_url object.email 48 %}" />
+  <img class="ui image"
```

Comment:
I think you might want the same `micro rounded` classes here:

```suggestion
  <img class="ui micro rounded image"
```

I'd check the social account listing template and maybe try what is there too.

Reply:
Looks like we aren't using rounded images on projects listings

https://github.com/readthedocs/ext-theme/blob/c91ab2b841b822ef4313b6ee4ccf67f17dee343f/readthedocsext/theme/templates/projects/includes/project_image.html#L12-L12

default_if_none only works if the value is none, in this case is undefined, so the default isn't applied.

Reply:
Yup, project and org listing views are square images, it's only the details/header image that _are_ rounded.